src/crystal/system/win32/file.cr

```diff
@@ -212,8 +212,8 @@ module Crystal::System::File
     end
   end
 
-  private def system_truncate(size : Int) : Nil
-    if LibC._chsize(fd, size) != 0
+  private def system_truncate(size) : Nil
```

Comment:
Why is the `Int` restriction gone?

Reply:
That's because I'm a newbie about crystal and I didn't know that `Int` was the base type of all integer types.

I thought that `Int` was 32-bit by analogy with C and changed the restriction from `Int` to `Int64` at first because the `size` argument is passed to the `Int64` argument of `LibC._chsize_s()`. However I got an error by the following code.

```crystal
File.open("test-file.txt", "w") do |file|
  file << "123456789012345678901234567890"
  file.truncate(20)
end
```
```shell
$ ./crystal/.build/crystal build --cross-compile --target x86_64-pc-windows-msvc "$@" trunc-test.cr 
Showing last frame. Use --error-trace for full trace.

In crystal/src/file.cr:763:5

 763 | system_truncate(size)
       ^--------------
Error: no overload matches 'File#system_truncate' with type Int32

Overloads are:
 - Crystal::System::File#system_truncate(size : Int64)
```
I could not imagine the error reason. I checked the definition of `system_truncate` in `src/crystal/system/unix/file.cr` and found that it didn't have such restriction. So I removed the restriction and it worked. I made a pull request after that.

I just now checked that it worked with the `Int` restriction. Should I revert the change at the line and push a commit?

Reply:
Yes, please!

`Int` is not well named, it should be named `Integer`. I hope we can rename it in the future.

Reply:
I pushed 5c20b89. Is it Okay?

Reply:
Perfect, thank you!